bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tippy from 'tippy.js';
+import _ from 'lodash';
+import cx from 'classnames';
+
+const propTypes = {
+
```

Comment:
missing proptype

Reply:
dont forget to add a proptype for the children, since it will only work on single element children.